Flask_and_SQL

JuanCastroFuentes opened the pull request:
Conexión a database. Empiezo a hacer el script para flaskear la user request y devolver datos de la database.

JuanCastroFuentes committed:
Flask_and_SQL

Conexión a database. Empiezo a hacer el script para flaskear la user request y devolver datos de la database.

albertoecf reviewed (changes requested):
We should fix 

- DATA_BASES (it should not be included) 
- Rename web_api_files/libs for "utils" (this will requiere some code refactor in other files such us web_api_files/api_local.py 
`from libs.openmeteo import MarineOpenMeteoClient
`

JuanCastroFuentes committed:
changes_requested

Change folder name libs for utils.
Modify comments on scripts.
Move databases out of repository.
